### DISCUSSION TITLE
I'm not able to get my vercel commands to work on Vercel with Visual Studio Code

### DISCUSSION THREAD

**Discussion by caretakerx** [vercel/vercel#6999] [Help] [answered]
I'm not able to get my vercel commands to work on Vercel with Visual Studio Code

My vercel command shows an error in terminal.

I'm using Mac what would be the commands to host my vercel site.

**Comment by Pieparker** [comment]
If you're using a third-party VSCode-specific tool, you'll need to consult the respective author. Otherwise, please check out the README for bug bashing instructions (https://github.com/vercel/vercel#readme) or [our docs on installing the CLI](https://vercel.com/cli) and [using the CLI](https://vercel.com/docs/cli).

**Reply by caretakerx** [reply]
I've tried everything and I can't get Vercel to host my site.

**Comment by caretakerx** [comment, marked as answer]
When I run Vercel I get this command.
Vercel CLI 23.1.2
Error! request to https://api.vercel.com/v13/now/deployments failed, reason: socket hang up

**Reply by Pieparker** [reply]
Plenty of useful, related advice [over here](https://github.com/vercel/vercel/discussions/6314) for that!

**Comment by caretakerx** [comment]
I tried them and no alas.

**Reply by Pieparker** [reply]
So you tried `npm run clean && vercel dev` and `vercel dev --listen 3001`?

Perhaps try removing the CLI and giving [the latest release](https://github.com/vercel/vercel/releases) another try?

**Comment by caretakerx** [comment]
Any other ideas.

**Reply by Pieparker** [reply]
Unfortunately not, @caretakerx. I'd be looking at clean installs. Failing that, create an issue [here](https://github.com/vercel/vercel/issues) with all relevant information (steps you've tried, error messages verbatim, OS versions).

Just a head's up, it is poor etiquette to spam or "bump" a thread. Furthermore, replies should be used, instead of further answers.

**Comment by caretakerx** [comment]
Ok thx.

**Comment by caretakerx** [comment]
I’ve always gotten my vercel commands to work.

**Comment by caretakerx** [comment]
I tried the npm i -g vercel and it installed. None of the other commands after did.
